frameworks/ruby/rails6/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -ex
+set -o pipefail;
```

Comment:
The semi-colons for line ending in this file aren't needed. Can you remove each of them?